FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.Common/Notification/FailedJobNotification.cs

```diff
@@ -0,0 +1,87 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using MediatR;
+using Microsoft.Health.Fhir.Synapse.Common.Models.Jobs;
+using Newtonsoft.Json;
+
+namespace Microsoft.Health.Fhir.Synapse.Common.Notification
+{
+    public class FailedJobNotification : INotification
```

Comment:
The `FailedJobNotification` and `SuccessfulJobNotification` share many common properties, is it possible to set a common parent notification for them?